assets/js/app/llms-forms.js

```diff
@@ -378,12 +377,49 @@ LLMS.Forms = {
 			this.$states.html( '<option>&nbsp</option>' );
 			this.disable_field( this.$states );
 		} else {
+			var placeholder = this.get_state_placeholder( this.$states, country_code );
 			this.enable_field( this.$states );
 			this.$states.html( opts );
+			if ( '' !== placeholder ) {
+				this.set_state_placeholder( this.$states, placeholder );
+			}
 		}
 
 	},
 
+    /**
+	 * Get the state placeholder and/or replace with {REGION}
+	 *
+	 * @since [version]
+	 * 
+	 * @param {String} country_code Currently selected country code. 
+	 * @return {String} placeholder text if any has been set, otherwise returns default state/region. 
+	 */
+	get_state_placeholder: function( holder, country_code ) {
+
+		var placeholder;
+		
```

Comment:
check out how the `update_locale_info()` method handles accessing the `this.address_info` object. 

The current code will error with `Uncaught TypeError: this.address_info is undefined` if the address_info object isn't intitialized for some reason.

It's an unlikely event but we should handle it *anyway*